src/core/CorePool.ts

```diff
@@ -100,10 +101,29 @@ export class CorePool {
     tickUpper: number,
     amount: JSBI
   ): { amount0: JSBI; amount1: JSBI } {
-    // TODO
+    if (JSBI.lessThanOrEqual(amount, ZERO)) {
+      console.error("[Error]: mint amount should greater than 0");
```

Comment:
Let's change it to `assert` for now because we not only want error message printing in console but also need an `Error` to be throwed so that we can catch it in test cases.

Reply:
Got it!